js/renovation/ui/form/__tests__/form.test.tsx

```diff
@@ -37,15 +38,24 @@ describe('Form > Markup', () => {
       const form = mount<Form>(<Form {...props} />);
 
       const scrollable = form.find(Scrollable);
-      expect(scrollable.props()).toEqual({
+      expect(scrollable.props()).toEqual(expect.objectContaining({
         aria: { role: 'form' },
         bounceEnabled: false,
         classes: 'dx-form',
         direction: 'both',
         useKeyboard: false,
         useNative: !!useNativeScrolling,
         useSimulatedScrollbar: !useNativeScrolling,
-      });
+      }));
+    });
+
+    it('root layoutManager is rendered', () => {
+      const props = { scrollingEnabled: true, useNativeScrolling } as FormProps;
+      const form = mount<Form>(<Form {...props} />);
+
+      const layoutManager = form.find(FormLayoutManager);
+      expect(layoutManager.exists()).toEqual(true);
+      expect(layoutManager.props().isRoot).toEqual(true);
```

Comment:
isRoot is not used in code, are we need it in this PR?

Reply:
fixed